stock_shipment_composer/models/stock_shipment_composer_line.py

```diff
@@ -0,0 +1,37 @@
+# Copyright 2025 Quartile (https://www.quartile.co)
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl)
+
+from odoo import api, fields, models
+
+
+class StockShipmentComposerLine(models.Model):
+    _name = "stock.shipment.composer.line"
+    _description = "Stock Shipment Composer Line"
+
+    composer_id = fields.Many2one(
+        "stock.shipment.composer", required=True, ondelete="cascade"
+    )
+    move_id = fields.Many2one("stock.move", required=True)
+    picking_id = fields.Many2one(related="move_id.picking_id", string="Transfer")
+    quantity = fields.Float(required=True)
```

Comment:
Can you check if this causes an issue when a composer is created from the wizard?

```suggestion
    quantity = fields.Float(compute="_compute_quantity", store="True", readonly=False)
```

```.py
    @api.depends("move_id")
    def _compute_quantity(self):
        for rec in self:
            rec.quantity = rec.move_id.composer_unallocated_qty
```

Reply:
There is no issue when we use compute method.